main.go

```diff
@@ -31,26 +31,34 @@ func main() {
 
 	if len(flag.Args()) < 1 {
 		fmt.Printf("\n\nNo input file\n\n")
-		fmt.Printf("usage: %s path\n", os.Args[0])
+		if *template_flag {
+			fmt.Printf("usage: %s <path to template file> (optional: <path to variable file>)\n", os.Args[0])
```

Comment:
@vidriduch Unfortunatelly, this is not how the usage screens for unix-type of commands are formatted. I believe it should adhere as much as possible to the [POSIX standard](http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap12.html), which has since been [summarized](http://stackoverflow.com/a/9727046) in a more [digestible way](http://courses.cms.caltech.edu/cs11/material/general/usage.html).